tools/SourceKit/docs/Protocol.md

```diff
@@ -38,6 +38,11 @@ must be given either the path to a file (`key.sourcefile`), or some text
 (`key.sourcetext`). `key.sourcefile` is ignored when `key.sourcetext` is also
 provided.
 
+| Request Name | Request Key | Description |
+| -------------:|:------------|:------------|
+| `codecomplete` | `codecomplete` |  Given a file will open a code-completion session which can be filtered upon using `codecomplete.update`. Each session must be closed using `codecomplete.close`.|
```

Comment:
This is a description of `codecomplete.open`.  `codecomplete` just returns completions and does not filter or create a session.  Also, you could remove the word "upon" entirely.